docs/insomnia/chaining-requests.md

```diff
@@ -5,57 +5,40 @@ category: "Requests and Responses"
 category-url: requests-and-responses
 ---
 
-Insomnia provides the ability to extract values from the responses of other requests. This is commonly referred to as chaining because the dependency upon another request forms a link.
+Insomnia allows chaining requests, or the ability to extract values from the responses of other requests. Values are passed using [Template Tags](/insomnia/template-tags) and [Environment Variables](/insomnia/environment-variables).
 
-Request chaining may be difficult to understand at first, especially if the concepts of [Template Tags](/insomnia/template-tags) or [Environment Variables](/insomnia/environment-variables) are unfamiliar. I suggest first taking a look at those if you are having trouble getting started.
+The following are common use cases for chaining requests:
 
-First, let's take a look at some examples that chaining may be useful for.
+* use a token from a login response as an Authorization header
+* use the ID from POST `/items` in PUT `/items/{id}`
+* use an entire body from one response in another request
 
-## Common Uses
+## Select Response Tag
 
-To help explain request chaining, here are some common use cases:
+In an empty URL bar or following a URL symbol such as `/` or `=`, either press CTRL+Space to open the autocomplete dropdown or start typing "response".
```

Comment:
The auto-complete dropdown behavior in the URL bar is specific to the URL bar, but more importantly request chaining doesn't require you to add the response tag specifically to the URL bar, so I'd vote to not specify the URL bar specifically. Any text input where template tags & environment variables can be used, response chaining can also be used.